### ISSUE TITLE
React: Masthead - Menu options within the "profile menu"

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4232]
## Detailed description

On Voiceover iPhone/Safari 

Issue: 
1. There is no label for these menu options. Voiceover iPhone announces only as "menu item"
2. Voiceover identifies "Focus Sentinel" link before and after end of the menu options which is not visible. Upon activation of these links, the same page refreshes. This issue is found in "dotcom shell react staging" as well.

Expected :
1. Provide appropriate label
2. If the user is not expected to interact with this element, this should be hidden for the screen reader as well.

**Comment by andysherman2121** [comment]
Hey @Praveen-sr Can you retest this? I tried using the voice over on my iPhone and it worked accordingly. I read out each menu item in the profile correctly. It's possible this issue has been fixed since the story was first created

**Comment by Praveen-sr** [comment]
@andysherman2121 We have retested this issue in Masthead and dotcomshell both React Staging as well as React canary
Issue remains the same barring one change. Now Label is identified for "My IBM" option within the profiles menu.

**Comment by andysherman2121** [comment]
The dropdown component we are using is coming directly from the Carbon library so I’m unable to fix this accessibility issue. I opened up an issue on the Carbon repo. 

https://github.com/carbon-design-system/carbon/issues/7388

@RobertaJHahn We should move this issue out of this release because we don’t know exactly when carbon will fix it

**Comment by asudoh** [comment]
One option to address this is going to a custom menu instead of using Carbon overflow menu; Carbon overflow menu is based on ["floating menu"](https://github.com/carbon-design-system/carbon/blob/v10.23.0/packages/react/src/internal/FloatingMenu.js) that requires additional a11y considerations. Web Components version took the custom menu route, basically one based on UI shell nav menu.

**Comment by RobertaJHahn** [comment]
This fix is dependent on a fix from Carbon.  Moving to Blocked from Scheduling pipeline.

**Comment by RobertaJHahn** [comment]
Not stale.  Still waiting for the Carbon fix.
